### ISSUE TITLE
Unsupported attribute: ims error while fetching users

### ISSUE THREAD

**Issue by harshal-suryawanshi-sp** [salesforceidentity/scimv2-pilot#12]
Here below query tried from POSTMAN
scim/v2/Users?attributes=entitlements,preferredLanguage,addresses,displayName,nickName,timezone,roles,externalId,active,groups,userName,title,locale,photos,phoneNumbers,ims,emails,meta,name,id,userType
 
Error:
{
  "schemas": "[\"urn:ietf:params:scim:api:messages:2.0:Error\"]",
  "detail": "Unsupported attribute: ims",
  "status": 400
}

**Comment by iglazer** [comment]
Yes we do not support the IMS attribute. Are you seeing it in our Schema?

**Comment by harshal-suryawanshi-sp** [comment]
No, now we are not seeing ims in schema.
It was raised earlier when ims was in schema.
